src/_data/service_lines_tabber.yml

```diff
@@ -84,7 +84,7 @@ en:
         - Achieve value-driven prioritization
         - Innovate through experimentation
       details: Build MVPs, Proof of Concepts, and Prototypes; Create a high-level technical strategy and roadmap; Customer journey mapping; Value stream mapping; Service blueprint and design; Product analytics; Usability testing; AB Testing; User research
-      image-path: /assets/custom/img/service-lines/service-line-example/tabber-placeholder-a.svg
+      image-path: /assets/custom/img/software-delivery/tabber-decorations_tabber-product-strategy.svg
```

Comment:
I'm a bit surprise you're not using the `service-lines` directory, e.g. `/assets/custom/img/service-lines/software-delivery/tabber-decorations_tabber-product-strategy.svg`

Not a biggie just trying to get on the same page for consistency

Reply:
The image folders are still a bit untidy with softmod images in 4 different places. I'm going to do some clean up on a separate branch